### PR TITLE
Upgrade jhipster to v6.0.1

### PR DESCRIPTION
**Pull request by vraravam**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/jhipster.rb [hunk 1]
```diff
@@ -3,14 +3,14 @@
...
-  sha256 "153bfaf4e778a3797a22704d6a2d7630a1b4741131bc72352bacd075c51fed24"
+  url "https://registry.npmjs.org/generator-jhipster/-/generator-jhipster-6.0.1.tgz"
+  sha256 "a1f8d9d91fbc77f46209b5023f41a50fccf3bb31d7de3d3d4ae8ca667530bd21"
 
   bottle do
     cellar :any_skip_relocation
-    sha256 "b6b825762f8c2fca4bf93bd5ff250cbbe0b1c30774ca90d9128a47595f2b9049" => :mojave
-    sha256 "eed470613019026525142d32f5ac228c909dafc5ab3dae2ccce7abff6f5587d3" => :high_sierra
-    sha256 "cba4b884da504ce15abaac2e692c59fa50068282d3e38ae5ec96756dafec0386" => :sierra
+    # sha256 "b6b825762f8c2fca4bf93bd5ff250cbbe0b1c30774ca90d9128a47595f2b9049" => :mojave
```
Comment (lembacon) [inline Formula/jhipster.rb]: Please leave the `bottle do` block untouched, it will be updated automatically upon merging. Thanks!

**Review by lembacon — CHANGES_REQUESTED** [review]

File: Formula/jhipster.rb (hunk 1 above)
Comment (vraravam) [reply Formula/jhipster.rb]: @lembacon - thanks (I didnt know that). Will commit with the changes reverted.

Comment (vraravam) [reply Formula/jhipster.rb]: @lembacon - done with the new commit. Please review once again.

**Review by lembacon — APPROVED** [review]

**Comment by lembacon** [comment]
Thanks @vraravam!

**Pull request closed without merging**